.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: false
```

Comment:
It's not how old your repo is, it's for how long it's been enabled on Travis. If you don't know that, then it's safe to use `sudo: false`. But it's likely we do know which repos were enabled in 2015 and which were enabled before.